diff --git a/src/minimuas_GCS.cpp b/src/minimuas_GCS.cpp
index 16ba63a..30a32d4 100644
--- a/src/minimuas_GCS.cpp
+++ b/src/minimuas_GCS.cpp
@@ -50,7 +50,7 @@ main(int argc, char **argv)
     std::string identity = argv[1];
     int interval_in_ms = std::stoi(argv[2]);
     int count = std::stoi(argv[3]);
-    std::string conf_dir = "/usr/local/bin";
+    std::string conf_dir = "@minimuas-files@/config";
     int delay = 0000;
 
     int iuas_sensor_idx = 0;
diff --git a/src/minimuas_GCS_shell.cpp b/src/minimuas_GCS_shell.cpp
index 8cec112..62955fd 100644
--- a/src/minimuas_GCS_shell.cpp
+++ b/src/minimuas_GCS_shell.cpp
@@ -49,7 +49,7 @@ int main(int argc, char **argv)
     }
 
     std::string identity = argv[1]; // Node's NDN name from input args
-    std::string conf_dir = "/usr/local/bin"; // config directory
+    std::string conf_dir = "@minimuas-files@/config"; // config directory
     int iuas_sensor_idx = 0; // hard-coded stuff
 
     ndn::Face m_face; // create an application face
diff --git a/src/minimuas_IUAS.cpp b/src/minimuas_IUAS.cpp
index 66a661d..0b7d0a4 100644
--- a/src/minimuas_IUAS.cpp
+++ b/src/minimuas_IUAS.cpp
@@ -78,7 +78,7 @@ main(int argc, char **argv)
     auto m_offboard = mavsdk::Offboard{system};
 
     std::string identity = argv[1];
-    std::string conf_dir = "/usr/local/bin";
+    std::string conf_dir = "@minimuas-files@/config";
 
     // Defining hard-coded sensor parameters
     auto sensor_idx = 0;
diff --git a/src/minimuas_WUAS.cpp b/src/minimuas_WUAS.cpp
index 2999e86..9640a58 100644
--- a/src/minimuas_WUAS.cpp
+++ b/src/minimuas_WUAS.cpp
@@ -67,7 +67,7 @@ main(int argc, char **argv)
     m_telemetry->set_rate_gps_info(0.5);
 
     std::string identity = argv[1];
-    std::string conf_dir = "/usr/local/bin";
+    std::string conf_dir = "@minimuas-files@/config";
 
     // Same as minimuas_GCS_shell
     ndn::Face m_face;
diff --git a/src/service-controller.cpp b/src/service-controller.cpp
index 4cbb273..6507506 100644
--- a/src/service-controller.cpp
+++ b/src/service-controller.cpp
@@ -12,7 +12,7 @@
 int
 main(int argc, char** argv)
 {
-  std::string conf_dir = "/usr/local/bin";
+  std::string conf_dir = "@minimuas-files@/config";
 
   try {
     ndn::Face m_face;
